### PR TITLE
OPCT-226: Added check rules for etcd

### PR DESCRIPTION
**Pull request by mtulio**
Introducing etcd check rules based in the parsed logs for etcd (etcd request took too long).

The acceptance values are calibrated from the baseline providers:

- AWS (ocp414rc0_AWS_None_202309222127_sonobuoy_47efe9ef-06e4-48f3-a190-4e3523ff1ae0.tar.gz)

Screenshot from 2023-09-25 13-41-29

- AWS (4.13.9-20230925-HighlyAvailable-aws-None-202309250459_sonobuoy_f4e06587-e7b3-4cbd-bcf4-d1350ec35d9a.tar.gz):

Screenshot from 2023-09-25 13-38-17

- vSphere (4.13.9-20230821-HighlyAvailable-vsphere-None.tar.gz):

Screenshot from 2023-09-25 13-34-52

The check rules is implemented in the feature https://github.com/redhat-openshift-ecosystem/provider-certification-tool/pull/76 
https://github.com/redhat-openshift-ecosystem/provider-certification-tool/blob/dc2156317dfc92b47c582b494fef47876337d94b/internal/opct/report/checks.go#L321-L388

### CODE REVIEW

**Comment by mtulio** [comment]
need to be merged after fixing broken master https://github.com/redhat-openshift-ecosystem/provider-certification-tool/commits/main
/hold

File: docs/review/rules.md [hunk 1]
```diff
@@ -163,6 +163,68 @@ Explore the pods:
...
+References:
+
+- [etcd: Hardware recommendations](https://etcd.io/docs/v3.5/op-guide/hardware/)
+- [OpenShift Docs: Planning your environment according to object maximums](https://docs.openshift.com/container-platform/4.11/scalability_and_performance/planning-your-environment-according-to-object-maximums.html)
+- [OpenShift KCS: Backend Performance Requirements for OpenShift etcd](https://access.redhat.com/solutions/4770281)
+- [IBM: Using Fio to Tell Whether Your Storage is Fast Enough for Etcd](https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd)
+
+
+### OPCT-011 <a name="OPCT-011"></a>
+- **Name**: etcd logs: slow requests: average should be under 1500ms
```
Comment (rvanderp3) [inline docs/review/rules.md:221]: 1500ms seems like a long time, particularly for the average.  I would think this would be closer to 150ms.

Comment (mtulio) [reply docs/review/rules.md:221]: @rvanderp3 the description is wrong. This should be the maximum! Good catch! :) 

WDYT for 1.5s for the maximum?

Comment (mtulio) [reply docs/review/rules.md]: fixed to max to 1s as per Slack conversation.
Description also fixed!

File: docs/review/rules.md [hunk 2]
```diff
@@ -163,6 +163,70 @@ Explore the pods:
...
+- [etcd: Hardware recommendations](https://etcd.io/docs/v3.5/op-guide/hardware/)
+- [OpenShift Docs: Planning your environment according to object maximums](https://docs.openshift.com/container-platform/4.11/scalability_and_performance/planning-your-environment-according-to-object-maximums.html)
+- [OpenShift KCS: Backend Performance Requirements for OpenShift etcd](https://access.redhat.com/solutions/4770281)
+- [IBM: Using Fio to Tell Whether Your Storage is Fast Enough for Etcd](https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd)
+
+
+### OPCT-011 <a name="OPCT-011"></a>
+
+- **Name**: etcd logs: slow requests: maximum should be under 1000ms
+- **Description**: etcd logs are reporting slow requests with average above 1000 milisseconds.
```
Comment (rvanderp3) [inline docs/review/rules.md:224]:
```suggestion
- **Description**: etcd logs are reporting slow requests with maximum above 1000 milisseconds.
```

**Comment by mtulio** [comment]
Thanks @rvanderp3 ! Fixed
/hold cancel

**Pull request merged**